infrastructure/cdn-in-a-box/traffic_ops/to-access.sh

```diff
@@ -168,85 +172,68 @@ to-enroll() {
 	else
 		export MY_CDN="CDN-in-a-Box"
 	fi
+	if [[ ! -z "$3" ]]; then
+		export MY_CACHE_GROUP="$3"
+	else
+		export MY_CACHE_GROUP="CDN_in_a_Box_Edge"
+	fi
+	if [[ ! -z "$4" ]]; then
+		export MY_TCP_PORT="$4"
+	else
+		export MY_TCP_PORT="80"
+	fi
+	if [[ ! -z "$5" ]]; then
+		export MY_HTTP_PORT="$5"
+	else
+		export MY_HTTP_PORT="443"
```

Comment:
Shouldn't this be e.g. `MY_HTTPS_PORT`? The field it's going into is `httpsPort`, not `httpPort`. It's confusing that we have `tcpPort` and `httpsPort` instead of `httpPort` or `tlsPort`, but we should at least be consistently confusing in the same way :P